### ISSUE TITLE
Test simulator submission

### ISSUE THREAD

**Issue by jonrkarr** [biosimulators/Biosimulators#5]
## Instructions

Please use this issue to contribute a containerized simulation tool, contribute another version of a simulation tool, or update a version of a simulation tool. Specifically, edit the YAML syntax between the `---` below and then click the `Submit new issue` button.

Instructions for containerizing simulation tools and examples are available at https://biosimulators.org/help.

## Pre-submission checklist

**Is your containerized simulation tool compliant with BioSimulators' conventions?**

- [ ] The container provides a [BioSimulators-compliant command-line interface](https://biosimulators.org/help) that supports the [COMBINE/OMEX archive](https://combinearchive.org/) and [SED-ML](https://sed-ml.org) formats
- [ ] The entrypoint of the container is set to this command-line interface
- [ ] The specifications of the simulator follow the [BioSimulators format](https://api.biosimulators.org)
- [ ] Optional, the source code for the simulation tool is publicly available
- [ ] Optional, the simulation tool is available through a software repository such as CRAN or PyPI
- [ ] Optional, examples, tutorials, and documentation for the simulation tool are publicly available
- [ ] Optional, the simulation tool is registered with [bio.tools](https://bio.tools/)
- [ ] Optional, the container uses [labels to provide BioContainers-compliant metadata](https://biosimulators.org/help)
- [ ] Optional, the container is registered with [BioContainers](https://biocontainers.pro/)

## Submission

**What is the name, version, and URL of the specifications of your simulation tool?**
Please follow the template YAML syntax below.

---
name: COPASI
version: 4.27.214
specificationsUrl: https://raw.githubusercontent.com/biosimulators/Biosimulators_COPASI/master/properties.json

---

## Post-submission approval workflow

Once you submit this issue, BioSimulators will use the following workflow to validate, approve, and commit your simulator to the BioSimulators database:

1. An issue will be created with the label `Submit simulator`.
2. An automated action will review your submission:
   1. Retrieve the specifications of your simulator.
   2. Validate these specifications.
   3. Validate the containerized simulator specified in your specifications.
3. If the validation fails, any errors will be posted to this issue. After fixing these issues, please edit the first block of this issue to re-initiate the automated validation.
4. If the validation succeeds, the `Validated` label will be added to the issue.
5. The BioSimulations Team will review your simulator.
6. If there are any concerns about your submission, the BioSimulations Team will use this issue to communicate with you about resolving them.
7. Once your simulator is approved, the BioSimulations Team will add the label `Approved` to this issue. 
8. The simulator will automatically be committed to the BioSimulations database and the issue will automatically be closed.

## Re-submission change list

If you need to revise your simulator to pass the validation, please use this section to briefly describe the changes that you made.

- debugged biosimulations_utils
- debugged again again again again again again agian again again again again again again agian gain

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators registry of containerized simulation tools! We will review your submission and discuss any issues here.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Your containerized simulator is valid!

**Comment by biosimulators-daemon** [comment]
A member of the BioSimulators team will review your submission before final committment to the registry.

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators registry of containerized simulation tools! We will review your submission and discuss any issues here.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators registry of containerized simulation tools! We will review your submission and discuss any issues here.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators registry of containerized simulation tools! We will review your submission and discuss any issues here.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Your container passed 0 test cases.

**Comment by biosimulators-daemon** [comment]
```diff
- No test cases are applicable to your container. Please use this issue to share appropriate test COMBINE/OMEX files with the BioSimulators Team.
```

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators registry of containerized simulation tools! [Action 324762616](https://github.com/biosimulators/Biosimulators/actions/runs/324762616) is reviewing your submission. We will discuss any issues with your submission here.

**Comment by biosimulators-daemon** [comment]
```diff
- Sorry. We encountered an unexpected error. Our team will review the error.
- 
-   get_data_in_issue() missing 1 required positional argument: 'issue'
```

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators registry of containerized simulation tools! [Action 324764972](https://github.com/biosimulators/Biosimulators/actions/runs/324764972) is reviewing your submission. We will discuss any issues with your submission here.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
```diff
- Sorry. We encountered an unexpected error. Our team will review the error.
- 
-   No module named 'docker'
```

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators registry of containerized simulation tools! [Action 324767117](https://github.com/biosimulators/Biosimulators/actions/runs/324767117) is reviewing your submission. We will discuss any issues with your submission here.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Your container passed 0 test cases.

**Comment by biosimulators-daemon** [comment]
```diff
- No test cases are applicable to your container. Please use this issue to share appropriate test COMBINE/OMEX files for the BioSimulators test suite. The BioSimulators Team will add these files to the test suite and then re-review your simulator.
```

**Comment by biosimulators-daemon** [comment]
[Action 324774723](https://github.com/biosimulators/Biosimulators/actions/runs/324774723) is committing your submission to the BioSimulators registry.

**Comment by biosimulators-daemon** [comment]
Your submission was committed to the BioSimulators registry. Thank you!

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators registry of containerized simulation tools! [Action 324861380](https://github.com/biosimulators/Biosimulators/actions/runs/324861380) is reviewing your submission. We will discuss any issues with your submission here.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Your container passed 0 test cases.

**Comment by biosimulators-daemon** [comment]
```diff
- No test cases are applicable to your container. Please use this issue to share appropriate test COMBINE/OMEX files for the BioSimulators test suite. The BioSimulators Team will add these files to the test suite and then re-review your simulator.
```

**Comment by biosimulators-daemon** [comment]
[Action 324774723](https://github.com/biosimulators/Biosimulators/actions/runs/324774723) is committing your submission to the BioSimulators registry.

**Comment by biosimulators-daemon** [comment]
Your submission was committed to the BioSimulators registry. Thank you!

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators registry of containerized simulation tools! [Action 324868228](https://github.com/biosimulators/Biosimulators/actions/runs/324868228) is reviewing your submission. We will discuss any issues with your submission here.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Your container passed 0 test cases.

**Comment by biosimulators-daemon** [comment]
[Action 324892027](https://github.com/biosimulators/Biosimulators/actions/runs/324892027) is committing your submission to the BioSimulators registry.

**Comment by biosimulators-daemon** [comment]
Your submission was committed to the BioSimulators registry. Thank you!

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators registry of containerized simulation tools! [Action 324892185](https://github.com/biosimulators/Biosimulators/actions/runs/324892185) is reviewing your submission. We will discuss any issues with your submission here.

**Comment by biosimulators-daemon** [comment]
```diff
- Sorry. We encountered an unexpected error. Our team will review the error.
- 
-   'dict' object has no attribute 'name'
```

**Comment by biosimulators-daemon** [comment]
[Action 324893404](https://github.com/biosimulators/Biosimulators/actions/runs/324893404) is committing your submission to the BioSimulators registry.

**Comment by biosimulators-daemon** [comment]
Your submission was committed to the BioSimulators registry. Thank you!

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators registry of containerized simulation tools! [Action 324901917](https://github.com/biosimulators/Biosimulators/actions/runs/324901917) is reviewing your submission. We will discuss any issues with your submission here.

**Comment by biosimulators-daemon** [comment]
```diff
- Sorry. We encountered an unexpected error. Our team will review the error.
- 
-   'dict' object has no attribute 'name'
```

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators registry of containerized simulation tools! [Action 324906248](https://github.com/biosimulators/Biosimulators/actions/runs/324906248) is reviewing your submission. We will discuss any issues with your submission here.

**Comment by biosimulators-daemon** [comment]
```diff
- Sorry. We encountered an unexpected error. Our team will review the error.
- 
-   'Submit simulator' is not a valid IssueLabel
```

**Comment by jonrkarr** [comment]
resettted

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators registry of containerized simulation tools! [Action 324913611](https://github.com/biosimulators/Biosimulators/actions/runs/324913611) is reviewing your submission. We will discuss any issues with your submission here.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Your container passed 0 test cases.

**Comment by biosimulators-daemon** [comment]
```diff
- Sorry. We encountered an unexpected error. Our team will review the error.
- 
-   here
```

**Comment by jonrkarr** [comment]
reset

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators registry of containerized simulation tools! [Action 324917342](https://github.com/biosimulators/Biosimulators/actions/runs/324917342) is reviewing your submission. We will discuss any issues with your submission here.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Your container passed 0 test cases.

**Comment by biosimulators-daemon** [comment]
```diff
- No test cases are applicable to your container. Please use this issue to share appropriate test COMBINE/OMEX files for the BioSimulators test suite. The BioSimulators Team will add these files to the test suite and then re-review your simulator.
```

**Comment by jonrkarr** [comment]
rest

**Comment by biosimulators-daemon** [comment]
Thank you @jonrkarr for your submission to the BioSimulators registry of containerized simulation tools! [Action 324923405](https://github.com/biosimulators/Biosimulators/actions/runs/324923405) is reviewing your submission. We will discuss any issues with your submission here.

**Comment by biosimulators-daemon** [comment]
The specifications of your simulator is valid!

**Comment by biosimulators-daemon** [comment]
Your container passed 0 test cases.

**Comment by biosimulators-daemon** [comment]
```diff
- No test cases are applicable to your container. Please use this issue to share appropriate test COMBINE/OMEX files for the BioSimulators test suite. The BioSimulators Team will add these files to the test suite and then re-review your simulator.
```
